exporter/management/commands/flattener.py

```diff
@@ -27,42 +27,52 @@ class Command(BaseCommand):
     """
 
     def handle(self, *args, **options):
-        consume(callback, "flattener_init", decorator=decorator)
+        consume(callback, "flattener_init", ["flattener_init", "flattener_file"], decorator=decorator)
 
 
 def callback(state, channel, method, properties, input_message):
     job_id = input_message.get("job_id")
-    max_rows_lower_bound = Job.objects.get(id=job_id).get_max_rows_lower_bound()
-
+    file_path = input_message.get("file_path")
     export = Export(job_id, export_type="flat")
```

Comment:
We can move this line into the `else` branch below. We can also extract the `else` branch into a `publish_file` method, so that each branch has a consistent format.